services/static-webserver/client/source/class/osparc/data/model/Node.js

```diff
@@ -307,6 +307,10 @@ qx.Class.define("osparc.data.model.Node", {
       return bootModeSB;
     },
 
+    getMinVisibleInputs: function(metaData) {
+      return ("minVisibleInputs" in metaData) ? metaData["minVisibleInputs"] : null;
```

Comment:
Similar to what we did for the `boot-options`. This will be sent over from the backend as `min-visible-inputs`.

Reply:
exactly 👌